pyiron_continuum/fenicsx/job/generic.py

```diff
@@ -191,7 +192,21 @@ def plot_mesh(self, V1):
         return PlotMesh.plotMesh(V1)
 
     def plot_deformed_functionspace(self, V1, uh, factor):
```

Comment:
It's actually not a problem that you caused, but is there a possibility to add doc strings? If you don't know, it's ok